.github/workflows/govmomi-govc-tests.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: ["1.22"]
+        go-version: ["1.23"]
```

Comment:
You could define this as a variable for the repo to avoid simplify updates.

```yaml
go-version: ${{ fromJson(vars.GO_VERSION_MATRIX) }}
```

or

if just one...

```yaml
go-version: ${{ fromJson(vars.GO_VERSION) }}
```